vita3k/gui/src/imgui_impl_sdl.cpp

```diff
@@ -137,6 +137,14 @@ IMGUI_API void ImGui_ImplSdl_NewFrame(ImGui_State *state) {
     int display_w, display_h;
     SDL_GetWindowSize(state->window, &w, &h);
     ImGui_ImplSdl_GetDrawableSize(state, display_w, display_h);
+#if __APPLE__
+    // On macOS, if HiDPI is enabled(SDL_WINDOW_ALLOW_HIGHDPI) SDL will create a window that size is 960x544 points.
+    // But it is actually 1920x1088 pixels. Since SDL_GetWindowSize returns 960x544 points, we need to multiply by 2.
+    int scale = display_w / w;
+    w *= scale;
+    h *= scale;
+#endif
```

Comment:
This doesn't make a lot of sense, io.DisplaySize should be the display size in points while io.DisplayFramebufferScale should be the scaling to account for highDPI (the doc explicitly says that DisplayFramebufferScale is for Retina screens)

Reply:
https://github.com/ocornut/imgui/commit/5a3f82e2f4f30e529f765d38aaa76866653b245a

Since we are using a custom renderer for SDL, manual scaling looks correct to me